src/dmd/traits.d

```diff
@@ -140,6 +140,7 @@ shared static this()
         "getFunctionAttributes",
         "getFunctionVariadicStyle",
         "getParameterStorageClasses",
+        "getCppNamespaces",
```

Comment:
I forgot to put this here in the original PR

Reply:
Should be its own PR.

Reply:
removed